client/reader/index.js

```diff
@@ -95,6 +96,20 @@ export default async function () {
 			clientRender
 		);
 
+		// User stream
+		if ( config.isEnabled( 'reader/user-profile' ) ) {
+			page(
+				'/read/users/:user_id',
```

Comment:
Should we use `:username` instead of `:user_id` which is more user friendly?

Reply:
Example: In browser history it's better to have the url with `username` instead of `user_id` which is more convenient if need a revisit.